backend-flask/README.txt

```diff
@@ -0,0 +1,70 @@
+INSTALL REQUIREMENTS
+You can install flask and other requirements such as RESTfulapi for flask by simply running this script:
+
+$ pip install -r requirements.txt
+
+-------------------------
+VENV
+To install venv enter the following command:
+
+$ python3 -m venv venv
```

Comment:
You have to specify a version, 'python3', when installing venv otherwise you will be met with an error:

"$ python -m venv venv
/usr/bin/python: No module named venv"

While running a python.py file you dont need to specify version so the '3' is not required. You can but you dont need to. But, for consistency I will update the README.